frontend/src/Pages/Student/Notice/Notices .jsx

```
@@ -0,0 +1,76 @@
+        } else {
+            setFilteredNotices(notices.filter(notice => notice.department_name === department));
+        }
+    };
+
+    return (
+        <div>
+            <h2 className="mb-4 text-lg font-bold">Notices</h2>
+
+            {/* Department Filter Dropdown */}
+            <div className="mb-4">
+                <label className="mr-2 font-semibold">Filter by Department:</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)